client/webserver/locales/en-us.go

```diff
@@ -0,0 +1,157 @@
+package locales
+
+var EnUS = map[string]string{
```

Comment:
In my opinion these need to be ordered. Take for example https://github.com/decred/decrediton/blob/master/app/i18n/translations/original.json which is ordered by identifier or even  https://github.com/planetdecred/dcrandroid/blob/master/app/src/main/res/values/strings.xml which is ordered by section.

Also, keys are all over the place. I like lower case underscored keys... But anything uniform would be good.

Reply:
I can organize by template.

Using the English text itself as the key has the benefit that the templates can still be read without a lookup. I've only shortened the key when the text is really long.

Reply:
I'm not opposed to going to all custom keys if there's a good reason, but I just wanted to point out the benefit of doing it this way.